CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+- Add a custom Signer for Hardware Wallets.
```

Comment:
"Add custom Harware Wallet Signer`HwiSigner` in `src/wallet/harwaresigner/` module". 

Or something along this line to elaborate a bit on it would be helpful.